### DISCUSSION TITLE
Add Boss key ( minimize to tray bar )

### DISCUSSION THREAD

**Discussion by nyannoying1337** [ppy/osu#16501] [Ideas] [answered]
osu! stable has the ability to minimize the game to the tray bar with 1 button press. Having this back in lazer would be neat.

![image](https://user-images.githubusercontent.com/17437302/149929720-db4433a6-3d3a-422a-9b69-42063f6b3554.png)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/6790
